.github/workflows/release.yml

```diff
@@ -159,14 +169,31 @@ jobs:
         source ./sourceme.sh && optic_build_for_release
     - run: echo "//registry.npmjs.org/:_authToken=${{ secrets.PUBLISH_NPM_OPTIC_BOT }}" > ~/.npmrc
     - run: node ./workspaces/scripts/publish.js
+      if: false
       env:
         OPTIC_PUBLISH_SCOPE: public
+    - name: Set is_prerelease output
+      id: release_type
+      run: |
+        OPTIC_VERSION=$(node -e "console.log(require('./workspaces/local-cli/package.json').version)")
+        if [[ $OPTIC_VERSION == *'-'* ]]; then
+          echo "v$OPTIC_VERSION is a prerelease"
+          echo "::set-output name=is_prerelease::true"
+        else
+          echo "v$OPTIC_VERSION is not a prerelease"
+          echo "::set-output name=is_prerelease::false"
+        fi
 
   # Creates debian package for users to install, hosted on s3
   release-deb:
     runs-on: ubuntu-latest
     needs: release-npm
+    if: needs.release-npm.outputs.is_prerelease != 'true'
```

Comment:
i cant help but feel like this is backwards. prereleases should be the default, and a public release is the less frequent case. this expressions reads better without negation, `if: needs.release-npm.outputs.is_release == 'true'`

Reply:
> prereleases should be the default, 

Yeah I reckon that makes sense as being the safer option.

Reply:
Had a look at this, but it reading `is_release` is more confusing to me: a prerelease is a kind of release, and "public" release doesn't cover it either, as a prerelease is also done publicly. I've changed the logic ever so slightly to get rid of the negation (in https://github.com/opticdev/optic/pull/408/commits/6a3f17eb265e32b14681c0b9455b502d4661f436), but that's where I'm going to leave it.